syncstorage/views/validators.py

```diff
@@ -303,12 +304,13 @@ def parse_multiple_bsos(request):
             logger.info(logmsg, userid, collection, id, msg, bso)
             continue
 
-        if count >= BATCH_MAX_COUNT:
```

Comment:
This was accidentally an inclusive range, because `enumerate` was starting the count at 0.

Reply:
To be clear, it's still an inclusive range now, it's just more explicitly inclusive no?

Reply:
Correct, it's now more obvious that it's an inclusive range on purpose.